vnext/src/Libraries/Pressability/Pressability.windows.js

```diff
@@ -0,0 +1,874 @@
+          }
+        }
+
+        if (this._isTouchWithinResponderRegion(touch, responderRegion)) {
+          this._receiveSignal('ENTER_PRESS_RECT', event);
+        } else {
+          this._cancelLongPressDelayTimeout();
+          this._receiveSignal('LEAVE_PRESS_RECT', event);
+        }
+      },
+
+      onResponderRelease: (event: PressEvent): void => {
+        this._receiveSignal('RESPONDER_RELEASE', event);
+      },
+
+      onResponderTerminate: (event: PressEvent): void => {
+        this._receiveSignal('RESPONDER_TERMINATED', event);
+      },
+
+      onResponderTerminationRequest: (): boolean => {
+        const {onResponderTerminationRequest} = this._config;
+        return onResponderTerminationRequest == null
+          ? true
+          : onResponderTerminationRequest();
+      },
+
+      onClick: (event: PressEvent) => {
+        const {onPress} = this._config;
+        if (onPress != null) {
+          onPress(event);
+        }
+      },
+    };
+
+    const mouseEventHandlers =
+      Platform.OS === 'ios' || Platform.OS === 'android'
+        ? null
+        : {
+            onMouseEnter: (event: MouseEvent): void => {
+              if (isHoverEnabled()) {
+                this._isHovered = true;
+                this._cancelHoverOutDelayTimeout();
+                const {onHoverIn, getHoverInDelayMS} = this._config;
+                if (onHoverIn != null) {
+                  const delay = getDelayMS(getHoverInDelayMS);
+                  if (delay > 0) {
+                    this._hoverInDelayTimeout = setTimeout(() => {
+                      onHoverIn(event);
+                    }, delay);
+                  } else {
+                    onHoverIn(event);
+                  }
+                }
+              }
+            },
+
+            onMouseLeave: (event: MouseEvent): void => {
+              if (this._isHovered) {
+                this._isHovered = false;
+                this._cancelHoverInDelayTimeout();
+                const {onHoverOut, getHoverOutDelayMS} = this._config;
+                if (onHoverOut != null) {
+                  const delay = getDelayMS(getHoverOutDelayMS);
+                  if (delay > 0) {
+                    this._hoverInDelayTimeout = setTimeout(() => {
+                      onHoverOut(event);
+                    }, delay);
+                  } else {
+                    onHoverOut(event);
+                  }
+                }
+              }
+            },
+          };
+
+    // [Windows
+    const keyboardEventHandlers = {
+      onKeyUp: (event: KeyEvent): void => {
+        if (
+          event.nativeEvent.code === 'Space' ||
+          event.nativeEvent.code === 'Enter' ||
+          event.nativeEvent.code === 'GamepadA'
+        ) {
+          const {onPressOut, onPress} = this._config;
+
+          // $FlowFixMe: PressEvents don't mesh with keyboarding APIs. Keep legacy behavior of pasing KeyEvents instead
+          onPressOut && onPressOut(event);
```

Comment:
This is terrible of course 😉 

All options involve some level of passing garbage unless we make upstream API changes unfortunately. This method of passing garbage is simple and the same as before though.